x/muxer/muxer.go

```diff
@@ -424,24 +424,12 @@ func (m *muxer) GetPluginInfo(ctx context.Context, e *emptypb.Empty) (*rpc.Plugi
 }
 
 func (m *muxer) Attach(ctx context.Context, req *rpc.PluginAttach) (*emptypb.Empty, error) {
-	attach := make([]func() error, len(m.servers))
-	for i, s := range m.servers {
-		s := s
-		attach[i] = func() error {
-			_, err := s.Attach(ctx, req)
```

Comment:
Why was this wrong for sub-providers to attach?

Reply:
It never actually attached a host to the muxer, as far as I could tell.

Reply:
update: it i=turns out this is a parallel functionality that doesn't affect m.host